chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -62,6 +62,12 @@ sealed abstract class Aggregate extends Data {
     }
   }
 
+  override def litOption = ???  // TODO implement me
```

Comment:
Is this TODO intended to be resolved in this PR or later?

Reply:
This is meant to be implemented later. Literal extractors on Bundles would be a new feature.